docs/src/main/asciidoc/Getting_Started_Guide.adoc

```diff
@@ -190,6 +190,52 @@ can install WildFly {wildflyVersion} on any operating system that supports the z
 tar formats. Refer to the Release Notes for additional information
 related to the release.
 
+[[installation_as_a_linux_service]]
+==== Installation as a Linux Service
+
+Once you have installed WildFly {wildflyVersion} from a Zip archive, you can create a Linux service so it can be started/stopped automatically when your system boots up or is shut down.
+
+The following steps describe the process to configure WildFly as a Systemd service. WildFly is shipped with a default Systemd Unit File configuration for Standalone and Domain mode. These wildfly-[standalone,domain].service files are located at the `$WFLY_HOME/bin/systemd` directory and assume that by default WildFly was installed at `/opt/wildfly` directory and would be launched with the `wildfly:wildfly` user:group. These files can be adjusted by you to cover your needs, for example, you can change the user that runs the service, the location of the WildFly installation you want to launch, the locations of the logs, and so on.
```

Comment:
BTW the correct stylization of 'systemd' is 'systemd'. Not systemD nor Systemd.

Reply:
Going above and beyond, e.g. our official RHEL docs use literal styling as well, thus `systemd`. 

https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/using_systemd_unit_files_to_customize_and_optimize_your_system/assembly_working-with-systemd-unit-files_working-with-systemd

Reply:
+1, we should be consistent and use "systemd".

Reply:
> +1, we should be consistent and use "systemd".

Or.. people mangle our project name all the time, should we mangle theirs in return? :-D